plugin/cmd/kube-scheduler/app/server.go

```diff
@@ -49,6 +49,7 @@ type SchedulerServer struct {
 	Address           util.IP
 	ClientConfig      client.Config
 	AlgorithmProvider string
+	AlgoProviders     string
```

Comment:
This is a little confusing to me. There are two fields, AlgorithmProvider and AlgoProviders.